faucet/privateKeysLoader.go

```diff
@@ -33,8 +33,8 @@ func NewPrivateKeysLoader(
 	if shardCoord == nil {
 		return nil, ErrNilShardCoordinator
 	}
-	if len(pemFileLocation) == 0 {
-		return nil, ErrInvalidPemFileLocation
+	if !core.DoesFileExist(pemFileLocation) {
```

Comment:
suggestion to have core.FileExists. Can be a Jira task for this in the refactoring user story

Reply:
created EN-10032